lib/dul_arclight/traject/ead2_config.rb

```diff
@@ -338,6 +340,12 @@
   to_field "#{selector}_teim", extract_xpath("/ead/archdesc/descgrp/#{selector}/*[local-name()!='head']")
 end
 
+to_field "para_tesim", extract_xpath("/ead/archdesc/descgrp/p", to_text: false)
```

Comment:
@gkostin1966 I don't think this XPath points to anything --- all the documents seem to have the structure between `<descgrp>` and `<p>`

```
    <descgrp type="admin">
      <acqinfo encodinganalog="541">
        <p> M-454 et al.</p>
      </acqinfo>
      <processinfo>
        <p>Cataloging funded by the National Endowment for the Humanities (NEH) and the "We the People" project.</p>
      </processinfo>
      <accessrestrict encodinganalog="506">
        <p>The collection is open for research.</p>
      </accessrestrict>
      <userestrict encodinganalog="540">
        <p> Copyright status is unknown</p>
      </userestrict>
      <prefercite encodinganalog="524">
        <p> African American History Collection, William L. Clements Library, The University of Michigan</p>
      </prefercite>
    </descgrp>
```

Reply:
@respinos Sadly not true.  I have xpaths for all the EADs attached to https://mlit.atlassian.net/browse/DCP2-339 and Clements has several.

Reply:
Example:
`<descgrp type="add"><p>In addition to this finding aid, the Clements Library has created the <title linktype="simple" href="https://clements.umich.edu/eadadd/wheeler_inventory.pdf" show="replace" actuate="onrequest">Henry M. Wheeler Photoprint Collection Inventory.</title>  This inventory lists items according to volume/box location and includes references to specific page/mat numbers, image descriptions (most of which are derived from captions originally inscribed by Wheeler on photograph versos), and photographic formats.</p></descgrp>`

Reply:
Ah - missed the `add`.

Reply:
@respinos umich-wcl-F-104-3whe

Reply:
this is now obsolete as I changed the approach